2025-01-07 11:37:38,583 - INFO - WARNING: logdir not provided. Results will not be saved to file.
2025-01-07 11:37:38,583 - INFO - -- BUILDING PRIOR START -------------
2025-01-07 11:37:38,584 - INFO - WARNING: Skipping invalid 'RelationalConstraint' with arguments {'targets': [], 'effectors': [], 'relationship': None}. Reason: Prior disabled.
WARNING: Skipping invalid 'InverseUnaryConstraint' with arguments {}. Reason: There are no inverse unary Token pairs in the Library.
WARNING: Skipping invalid 'DomainRangeConstraint' with arguments {}. Reason: Prior disabled.
WARNING: Skipping invalid 'LanguageModelPrior' with arguments {'weight': None}. Reason: Prior disabled.
WARNING: Skipping invalid 'MultiDiscreteConstraint' with arguments {'dense': False, 'ordered': False}. Reason: Prior disabled.
2025-01-07 11:37:38,584 - INFO - LengthConstraint: Sequences have minimum length 4.
                  Sequences have maximum length 30.
RepeatConstraint: [const] cannot occur more than 1 times.
TrigConstraint: [sin, cos] cannot be a descendant of [sin, cos].
ConstConstraint: [const] cannot be the only unique child of [sin, cos, add, sub, mul, div].
NoInputsConstraint: Sequences contain at least one input variable Token.
UniformArityPrior: Activated.
SoftLengthPrior: No description available.
2025-01-07 11:37:38,584 - INFO - -- BUILDING PRIOR END ---------------

2025-01-07 11:37:38,584 - INFO - WARNING: max_length (64) will be overridden by value from LengthConstraint (30).
2025-01-07 11:37:38,668 - INFO - WARNING: Entity <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:38,697 - INFO - WARNING: Entity <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:38,732 - INFO - WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f53140edb50>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f53140edb50>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,014 - INFO - WARNING: Entity <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,034 - INFO - WARNING: Entity <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,060 - INFO - WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f531408c390>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f531408c390>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,195 - INFO - WARNING: Entity <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method MultiRNNCell.call of <tensorflow.python.ops.rnn_cell_impl.MultiRNNCell object at 0x7f5316932350>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,215 - INFO - WARNING: Entity <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method LSTMCell.call of <tensorflow.python.ops.rnn_cell_impl.LSTMCell object at 0x7f5317bd2e10>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,241 - INFO - WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f530f52bb90>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f530f52bb90>>: AttributeError: module 'gast' has no attribute 'Index'
2025-01-07 11:37:39,765 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:37:46,746 - INFO - [00:00:00:06.98] Training iteration 1, current best R: 0.9397
2025-01-07 11:37:46,746 - INFO - 
	** New best
2025-01-07 11:37:46,746 - INFO - 	Reward: 0.939723283353037
2025-01-07 11:37:46,746 - INFO - 	Count Off-policy: 0
2025-01-07 11:37:46,746 - INFO - 	Count On-policy: 1
2025-01-07 11:37:46,746 - INFO - 	Originally on Policy: True
2025-01-07 11:37:46,746 - INFO - 	Invalid: False
2025-01-07 11:37:46,746 - INFO - 	Traversal: sin,mul,1.0242486263729234,sub,add,x1,x1,x1
2025-01-07 11:37:46,746 - INFO - 	Expression:
2025-01-07 11:37:46,935 - INFO - 	  sin(1.0242486263729234⋅x₁)

2025-01-07 11:37:46,937 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:37:51,464 - INFO - [00:00:00:04.53] Training iteration 2, current best R: 0.9525
2025-01-07 11:37:51,464 - INFO - 
	** New best
2025-01-07 11:37:51,464 - INFO - 	Reward: 0.9525127415160207
2025-01-07 11:37:51,464 - INFO - 	Count Off-policy: 0
2025-01-07 11:37:51,464 - INFO - 	Count On-policy: 1
2025-01-07 11:37:51,464 - INFO - 	Originally on Policy: True
2025-01-07 11:37:51,464 - INFO - 	Invalid: False
2025-01-07 11:37:51,464 - INFO - 	Traversal: sin,sub,sub,sub,x1,x1,x1,3.20349
2025-01-07 11:37:51,464 - INFO - 	Expression:
2025-01-07 11:37:51,471 - INFO - 	  -sin(x₁ + 3.20349)

2025-01-07 11:37:51,473 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:37:56,009 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:00,688 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:05,172 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:09,193 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:13,906 - INFO - [00:00:00:04.71] Training iteration 7, current best R: 0.9547
2025-01-07 11:38:13,906 - INFO - 
	** New best
2025-01-07 11:38:13,906 - INFO - 	Reward: 0.9547239796298319
2025-01-07 11:38:13,906 - INFO - 	Count Off-policy: 0
2025-01-07 11:38:13,906 - INFO - 	Count On-policy: 1
2025-01-07 11:38:13,906 - INFO - 	Originally on Policy: True
2025-01-07 11:38:13,906 - INFO - 	Invalid: False
2025-01-07 11:38:13,906 - INFO - 	Traversal: add,sin,x1,sin,div,x1,8.509507
2025-01-07 11:38:13,906 - INFO - 	Expression:
2025-01-07 11:38:13,912 - INFO - 	  sin(0.117515621057718⋅x₁) + sin(x₁)

2025-01-07 11:38:13,913 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:18,150 - INFO - -- RUNNING ITERATIONS START -------------
2025-01-07 11:38:22,507 - INFO - [00:00:00:04.36] Training iteration 9, current best R: 1.0000
2025-01-07 11:38:22,507 - INFO - 
	** New best
2025-01-07 11:38:22,507 - INFO - 	Reward: 1.0
2025-01-07 11:38:22,507 - INFO - 	Count Off-policy: 0
2025-01-07 11:38:22,508 - INFO - 	Count On-policy: 1
2025-01-07 11:38:22,508 - INFO - 	Originally on Policy: True
2025-01-07 11:38:22,508 - INFO - 	Invalid: False
2025-01-07 11:38:22,508 - INFO - 	Traversal: mul,0.5941976,add,sin,x1,sin,x1
2025-01-07 11:38:22,508 - INFO - 	Expression:
2025-01-07 11:38:22,510 - INFO - 	  1.1883952⋅sin(x₁)

2025-01-07 11:38:22,511 - INFO - [00:00:00:04.36] Early stopping criteria met; breaking early.
2025-01-07 11:38:22,956 - INFO - Invalid expressions: 7502 of 9000 (83.4%).
2025-01-07 11:38:22,956 - INFO - Error type counts:
2025-01-07 11:38:22,956 - INFO -   None: 7502 (100.0%)
2025-01-07 11:38:22,956 - INFO - Error node counts:
2025-01-07 11:38:22,956 - INFO -   None: 7502 (100.0%)
2025-01-07 11:38:22,957 - INFO - 1.1883952⋅sin(x₁)
2025-01-07 11:38:22,957 - INFO - Using time(s):
2025-01-07 11:38:22,957 - INFO - 44.500239610672
